src/expr.rs

```diff
@@ -1818,7 +1818,7 @@ const FORMAT_LIKE_WHITELIST: &[&str] = &[
     "eprintln!",
     "format!",
     "format_args!",
-    "panic!",
+    "print!",
```

Comment:
Expand this snippet below to see that `panic!` appears in the list further below. Looking deeper now, though, `panic!` belongs in the part of the list since it is part of the Standard Library. Shall I add it back and keep `print!` here?

Reply:
Yes please! And thank you very much for adding tests!